charts/super-agent-deployment/values.yaml

```diff
@@ -96,6 +96,10 @@ fedramp:
 # @default -- `false`
 verboseLog:
 
+# -- Enable the cleanup of super-agent managed resources when the chart is uninstalled.
+# If disabled, agents and / or agent configurations could remain in the cluster after the chart is uninstalled.
```

Comment:
I would specify that also the subchart has a similar mechanism

Reply:
I've just rephrased, let me know what you think